photon-ml/src/integTest/scala/com/linkedin/photon/ml/avro/ModelProcessingUtilsTest.scala

```diff
@@ -73,7 +73,68 @@ class ModelProcessingUtilsTest extends SparkTestUtils with TestTemplateWithTmpDi
     val numberOfOutputFilesForRandomEffectModel = 2
     val outputDir = getTmpDir
     val outputDirAsPath = new Path(outputDir)
-    ModelProcessingUtils.saveGameModelsToHDFS(gameModel, featureShardIdToFeatureNameAndTermToIndexMapMap, outputDir,
+    ModelProcessingUtils.saveGameModelsToHDFS(gameModel, indexMapLoaders, outputDir,
+      numberOfOutputFilesForRandomEffectModel, sc)
```

Comment:
If all of the parameters can't fit on one line, they should each be on a separate line.